platform/api/oci/api.go

```diff
@@ -120,3 +143,11 @@ func (a *API) GC(gracePeriod time.Duration) error {
 func boolToPtr(b bool) *bool {
 	return &b
 }
+
```

Comment:
do we not have these elsewhere to just import?

Reply:
We do not, I can certainly add a commit making these more general though.

Reply:
If we don't, I'd say leave it as is for now. We can refactor laterif we want; keep this PR on topic.